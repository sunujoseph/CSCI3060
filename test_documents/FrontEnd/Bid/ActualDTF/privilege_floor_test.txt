04 i t e m 4                 userSS better admin    500.01
00 better admin    AA 987654.32
