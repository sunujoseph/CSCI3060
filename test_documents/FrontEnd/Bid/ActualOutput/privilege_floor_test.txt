Enter Username: 
Enter Item: 
Enter Seller Username: 
Enter Bid: 
