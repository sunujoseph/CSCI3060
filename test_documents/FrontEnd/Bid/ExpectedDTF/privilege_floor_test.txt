04 i t e m 4           userSS          admin1          500.01
00 better admin    AA 987654.32