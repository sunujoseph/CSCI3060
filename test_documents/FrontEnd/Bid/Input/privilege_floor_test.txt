login
better admin
bid
i t e m 4
userSS
500.01
logout
