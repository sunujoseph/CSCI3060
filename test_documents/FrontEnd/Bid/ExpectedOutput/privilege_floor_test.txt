Enter Username: 
Enter Item Name: 
Enter Seller Username: 
Current Highest Bid Is: 100.00
Enter Bid: